integration_tests/mockApis/bodyScan.ts

```diff
@@ -30,7 +30,21 @@ export default {
     })
   },
 
-  stubGetBodyScanInfo: (details): SuperAgentRequest => {
+  stubGetBodyScan: ({ prisonNumber, details }: Record<string, number>): SuperAgentRequest => {
+    return stubFor({
+      request: {
+        method: 'GET',
+        urlPattern: `/bodyscan/body-scans/prisoners/${prisonNumber}`,
+      },
+      response: {
+        status: 200,
+        headers: { 'Content-Type': 'application/json;charset=UTF-8' },
+        jsonBody: details,
+      },
+    })
+  },
+
+  stubAddBodyScan: (details): SuperAgentRequest => {
```

Comment:
This is for retrieving lots of bodyscans for a list of prisoners.

could we rename it: `stubBulkGetBodyScan` or something?